requirements.txt

```diff
@@ -97,7 +97,6 @@ WebTest==2.0.24
 Werkzeug==0.11.15
 WTForms==2.1
 xvfbwrapper==0.2.9
--e .
```

Comment:
Ah this again :)

Won't this break our deploy, or at least my practiced way of doing things?